test/rest_api/test_storage_service.py

```diff
@@ -586,3 +594,20 @@ def test_range_to_endpoint_map_with_table_param(cql,  rest_api, tablets_enabled)
                 endpoint = entry["value"]
 
                 assert endpoint == expected_endpoint, f"Unexpected endpoint={endpoint} for token_range={token_range}. Expected endpoint was {expected_endpoint}"
+
+@pytest.mark.xfail(reason="rest_api suite doesn't support tablets yet (#17338), run test manually")
+def test_ownership_tablets_enabled_keyspace_param_used(cql, this_dc, rest_api):
+    with new_test_keyspace(cql, f"WITH REPLICATION = {{ 'class' : 'NetworkTopologyStrategy', '{this_dc}' : 1 }} AND TABLETS = {{ 'enabled': true }}") as keyspace:
+        resp = rest_api.send("GET", f"storage_service/ownership/{keyspace}")
+        assert resp.status_code == requests.codes.bad_request
+
+        resp_json = resp.json()
+        actual_error_reason = resp_json["message"]
+        expected_error_reason = f"storage_service/ownership is not supported for keyspace {keyspace} because it uses tablets."
+        assert expected_error_reason == actual_error_reason
+
+@pytest.mark.xfail(reason="rest_api suite doesn't support tablets yet (#17338), run test manually")
+def test_ownership_tablets_enabled_keyspace_param_missing(cql, this_dc, rest_api):
+    with new_test_keyspace(cql, f"WITH REPLICATION = {{ 'class' : 'NetworkTopologyStrategy', '{this_dc}' : 1 }} AND TABLETS = {{ 'enabled': true }}") as keyspace:
+        resp = rest_api.send("GET", f"storage_service/ownership")
+        resp.raise_for_status()
```

Comment:
I am not entirely sure about this test case. It passes, but there is a keyspace that uses tablets.

Should such request be also discarded like the one above?

Reply:
What does it return for the vnodes case?

Reply:
Array of pairs: (node: IP, ownership: float) is returned.

Currently, I do not see any difference in results when keyspace is not specified when using this API. 

This is a sample result for **_storage_service/ownership_**:
`[{'key': '127.7.51.50', 'value': '1.0000001'}]`

Reply:
Try a keyspace with RF=1 and initial_tablets=1 (when using tablets)

With vnodes, you should get node1->0.5 node2->0.5 (won't be exactly 0.5)
With tablets, you should get nodeX->0 nodeY->1 (can't predict if X is 1 or 2)

Reply:
btw reaching ownership > 1 is impressive